perf/lua/1mops_write.lua

```diff
@@ -11,7 +11,30 @@ local math = require('math')
 local json = require('json')
 local fio = require('fio')
 
-local params = require('internal.argparse').parse(arg, {
+local benchmark = require('benchmark')
+
+local USAGE = [[
```

Comment:
I propose to split usage for two parts: first part is autogenerated in `benchmark.lua` and contains short usage like `Usage: taskset 0xef tarantool 1mops_write.lua [options]` and a link to recommendations for benchmarking [^1], second part is test-specific, and it is a multiline comment with description of all supported options.


[^1]: https://github.com/tarantool/tarantool/wiki/Benchmarking

Reply:
I've moved the autogenerated part to the `benchmark.lua`. Also, I don't mention taskset in the generated part since it may not be the same for different benchmarks. Hence, I've added a link to our wiki with all the useful notes about the setup of the machine for benchmarking.